static/app/views/starfish/index.tsx

```diff
@@ -15,6 +19,13 @@ type Props = {
 const queryClient = new QueryClient();
 
 function StarfishContainer({organization, children}: Props) {
+  const parameterizedPathname = useStarfishParameterizedPathname();
+  useEffect(() => {
+    trackAnalytics('starfish.pageview', {
```

Comment:
@edwardgou-sentry why do we need this? We have this already: https://develop.sentry.dev/analytics/#route-based-frontend-analytics